core/storage/exploration/gae_models.py

```diff
@@ -402,6 +403,29 @@ def delete_multi( # type: ignore[override]
             ExplorationCommitLogEntryModel.update_timestamps_multi(
                 commit_log_models)
             datastore_services.put_multi(commit_log_models)
+        else:
+            # Delete the ExplorationVersionHistoryModels if force_deletion is
+            # True.
+            if force_deletion:
```

Comment:
Why do you need this `if`? The else is already only executed when the `force_deletion` is `True`

Reply:
Removed it now.